charts/opentelemetry-ebpf/values.yaml

```diff
@@ -0,0 +1,275 @@
+# Default values for opentelemetry-ebpf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+clusterName: ""
+
+images:
+  tag: "v0.10.0"
+  repository: otel
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+
+resources: {}
+
+otelCollector:
+  address: ""
+  port: ""
+
+log:
+  console: false
+  # possible values: { error | warning | info | debug | trace }
+  level: warning
+
+debug:
+  enabled: false
+  storeMinidump: false
+  sendUnplannedExitMetric: false
+
+kernelCollector:
+  enabled: true
+  serviceAccount:
+    create: true
+    name: ""
+  image:
+    repository: ""
+    tag: ""
+    name: opentelemetry-ebpf-kernel-collector
+
+  nodeSelector: {}
+  disableHttpMetrics: false
+
+  tolerations:
+    - operator: "Exists"
+      effect: "NoExecute"
+    - operator: "Exists"
+      effect: "NoSchedule"
+
+  affinity: {}
+  resources: {}
+
+  # uncomment the line below to disable automatic kernel headers fetching
+  # fetchKernelHeaders: false
+
+  # uncomment to enable enrichment using Docker metadata
+  # useDockerMetadata: true
+
+  # uncomment to enable enrichment using Nomad metadata (https://www.nomadproject.io/)
+  # collectNomadMetadata: true
+
+cloudCollector:
+  enabled: false
+  image:
+    repository: ""
+    tag: ""
+    name: opentelemetry-ebpf-cloud-collector
+
+  serviceAccount:
+    create: true
+    name: ""
+    annotations:
+      {}
+      ## eks.amazonaws.com/role-arn: "role-arn-name"
+
+  tolerations: []
+  affinity: {}
+
+k8sCollector:
+  enabled: true
+  serviceAccount:
+    create: true
+    name: ""
+  relay:
+    image:
+      repository: ""
+      tag: ""
+      name: opentelemetry-ebpf-k8s-relay
+  watcher:
+    image:
+      repository: ""
+      tag: ""
+      name: opentelemetry-ebpf-k8s-watcher
+
+  tolerations: []
+  affinity: {}
+
+reducer:
+  image:
+    repository: ""
+    tag: ""
+    name: opentelemetry-ebpf-reducer
+  extraArgs: {}
```

Comment:
In your design, we can add additional parameters to the reducer, such as 
```yaml
extraArgs:
  --prom=0.0.0.0:7010
```
 but there is no place to configure additional ports for our reducer service, so our 7010 cannot be accessed. Perhaps we can refer to `opentelemetry-collecotr` chart's  design:  https://github.com/open-telemetry/opentelemetry-helm-charts/blob/0def0da3d8beb5e95840e62c614fd616694c287e/charts/opentelemetry-collector/values.yaml#L240

Reply:
and we should add following values for `reducer`:

```yaml
service:
  # NodePort or LoadBalancer
  type: ClusterIP
```